src/main/java/edu/harvard/iq/dataverse/GuestbookResponseServiceBean.java

```diff
@@ -226,6 +238,20 @@ public void streamResponsesByDataverseIdAndGuestbookId(OutputStream out, Long da
         }
     }
     
+    private String formatPersistentIdentifier(String protocol, String authority, String identifier) {
+        // Note that the persistent id may be unavailable for this dvObject:
+        if (StringUtil.nonEmpty(protocol) && StringUtil.nonEmpty(authority) && StringUtil.nonEmpty(identifier)) {
+            StringBuilder sb = new StringBuilder();
+            sb.append(protocol);
+            sb.append(":");
+            sb.append(authority);
+            sb.append("/");
+            sb.append(identifier);
+            return sb.toString();
+        } 
+        return "N/A";
```

Comment:
I'm fine with "N/A" here if there is no PID. Heads up to @jggautier

Reply:
Thanks. I have no preference. Another conventional option is to leave it blank, right?

In the case where some files in a Dataverse repository have file PIDs and some files don't (which I don't think is the case for any Dataverse repository yet), would using "N/A" be better than leaving it blank? I'm trying to remember the details of another guestbook-related conversation we had with a repository admin who said she preferred using a value instead of leaving it blank because it was a clearer to her that a record was written and, i.e. that the code worked.

Reply:
@jggautier if you're fine with it, so am I. Thanks.